startup.py

```diff
@@ -28,7 +28,7 @@ class AliasLauncher(SoftwareLauncher):
     CODE_NAMES = {
         "AutoStudio": dict(flags="-a as", icon="icon_as_256.png"),
         "Surface": dict(flags="-a ss", icon="icon_ss_256.png"),
-        "Design": dict(flags="-a ds", icon="icon_cs_256.png"),
```

Comment:
I believe this icon has been removed as the product no longer exists.

Reply:
I didn't actually make that change -- it was from a merge with master. I will just remove that change from this branch